client/lib/media/utils/create-transient-media-id.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import impureLodash from 'lib/impure-lodash';
+
+const { uniqueId } = impureLodash;
+
```

Comment:
Is it necessary to use `impureLodash` in this instance? I think it's only for when we need to mock a lodash function for testing but we aren't doing that for this module, right?

I'm not sure it matters either way, I mostly just want to make sure my understanding is correct.

Reply:
It seems like some tests in `lib/media/utils/create-transient-media.js` were failing when using the regular version of lodash, that's why I copied the code over to that file and left it like it was. Not ideal. Maybe we should adjust the tests there. I'll follow up with a PR if applicable.